extensions/roc-plugin-repo/src/commands/utils/execute.js

```diff
@@ -0,0 +1,27 @@
+// A minimal version of execute existing in roc
+
+import { exec } from 'child_process';
+
+export default function(command) {
+  return new Promise((resolve, reject) => {
```

Comment:
We will still need to wrap `exec` in someway to make the callback into a promise structure, so I think we need to keep the promise creation here.